src/main/webapp/app/shared/util/firebase/firebase-review-utils.tsx

```diff
@@ -268,7 +268,9 @@ export const getCompactReviewInfo = <T extends BaseReviewLevel>(review: T): T =>
   }
   let childReview = review.children[0];
   if (childReview?.nestedUnderCreateOrDelete) {
-    return review;
+    if (isCreateReview(review)) {
+      return review;
+    }
   }
```

Comment:
I don't remember exactly why I changed this. However, in that PR that you pointed out I think I wanted to return review for isDeleteReview as well.

Reply:
If you cannot reproduce this (https://github.com/oncokb/oncokb-pipeline/issues/643#issuecomment-2617196177) issue, then feel free to merge